issue_test.go

```diff
@@ -1214,3 +1214,44 @@ func TestIssueService_GetWorklogs(t *testing.T) {
 		t.Errorf("Error given: %s", err)
 	}
 }
+
+func TestIssueService_GetWatchers(t *testing.T) {
```

Comment:
Would it be possible to Add unit tests for `RemoveWatcher` and `AddWatcher` also?